.github/workflows/android_agp8.yml

```diff
@@ -0,0 +1,40 @@
+name: Android CI with AGP8
+
+on:
+  push:
+    branches: [ "develop", "main" ]
+  pull_request:
+    branches: [ "develop", "main" ]
+
+jobs:
+  validateAGP8:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        agp: [ '8.0.0','8.0.1','8.0.2','8.1.0']
+
+    name: AGP Matrix Test - AGP ${{ matrix.agp }}
+    env:
+      VERSION_AGP: ${{ matrix.agp }}
```

Comment:
Ibid - not used. Is `VERSION_AGP` now recognized by the build files?

Reply:
FYI, if you really want to nerd-out on the matrix tests, we can provide pretty much any build config value as a command line arg. We currently recognize (for example): 
```
newrelic.agent.version=+
newrelic.agent.snapshot=
newrelic.agent.ndk=+
newrelic.agp.version=8.+
```